testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -68,6 +68,9 @@
 public abstract class BaseConnectorTest
         extends AbstractTestDistributedQueries
 {
+    private static final String MATERIALIZED_VIEW_NAME = "test_materialized_view_" + randomTableSuffix();
```

Comment:
Please move these variables into your test method.
Every derived class will get the same static variable and if tests for multiple connectors supporting MVs run in parallel, then there can be a clash.

Reply:
Wouldn't each connector have a different catalog?